### DISCUSSION TITLE
Fedora COPR Package

### DISCUSSION THREAD

**Discussion by solomspd** [oskarsh/Yin-Yang#192] [General]
I think having a fedora COPR package along side an arch AUR package would be nice and help easier installation. I am interested in spending some time on this and maintaining a COPR package for yin-yang.

I wanted to check there is interest and if there is anything I should be aware of.

**Comment by solomspd** [comment]
I have started [a COPR](https://copr.fedorainfracloud.org/coprs/solom/yin-yang/) where I can start experimenting with building it.

 Please let me know if the maintainers do not wish me to package it on COPR so I can take it down.

**Comment by l0drex** [comment]
That's great! If you have any questions or suggestions, please let me know!

For example, I am thinking about improving the installation process. The installation script is a bit hard to maintain, I would prefer to create an app manifest and let pip handle the rest. Or maybe I could compile everything into a single binary.

**Reply by beedell-roke** [reply]
> https://github.com/oskarsh/Yin-Yang/discussions/192#discussioncomment-5699810

@l0drex, having it on `pip` would do wonders for installation package maintainability (only one repo for dev to maintain package for) and for users, since it works literally everywhere. Is it currently installable from `pip3` without needing to clone the repo?

**Comment by gumbogal** [comment]
Any movement on this? I don't mind compiling every now and then but automatic updates would be excellent stuff. Thanks for your efforts regardless.

**Comment by beedell-roke** [comment]
https://github.com/baduhai/Koi/blob/894b3c03da50ceda7d866775eb937f08ae506283/README.md#fedora---copr provides https://copr.fedorainfracloud.org/coprs/birkch/Koi/.
